OpenRA.Mods.RA/SupportPowers/NukePower.cs

```diff
@@ -70,10 +71,12 @@ public override void Activate(Actor self, Order order, SupportPowerManager manag
 		{
 			base.Activate(self, order, manager);
 
-			if (self.Owner.IsAlliedWith(self.World.RenderPlayer))
-				Sound.Play(Info.LaunchSound);
-			else
-				Sound.Play(Info.IncomingSound);
+			var rp = self.World.RenderPlayer;
+			var sound = self.Owner.IsAlliedWith(rp) ? Info.LaunchSound : Info.IncomingSound;
+			var radars = self.World.ActorsWithTrait<ProvidesRadar>().Where(at => at.Actor.Owner == rp && at.Trait.IsActive);
```

Comment:
Ok.  If this is going to be completely rewritten, then it might make sense to close this and open a new one once the reimplementation is ready.